pcd_target_path: "/home/rm/sp_loc_25/pcd/map.pcd"
topic_name: "/cloud_registered"
#使用最好的结果，否则使用满足max_score或max_error的最后结果
use_best_result: true
max_dist_sq: 1.0
#高度高于height_threshold的点云不参与定位,单位米

height_flag: false  
height_threshold: 15
#[tx/m, ty/m, tz/m, yaw/度, pitch/度, roll/度]
init_params: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
voxel_grid_size: 0.02
#icp_point_to_plane
max_score: 0.05
